packages/text-block/src/TextBlock.tsx

```diff
@@ -23,7 +23,7 @@ export const TextBlock: FC<BlockProps> = ({ appBridge }) => {
     return (
         <div data-test-id="text-block">
             <RichTextEditor
-                designTokens={designTokens ?? undefined}
+                designTokens={designTokens}
```

Comment:
There are a few other blocks which maybe need this change?
- DoDontItem.tsx
- ImageCaption.tsx
- PersonalNoteBlock.tsx
- QuoteBlock.tsx

Reply:
Done 👍🏻